libs/spark/src/theme/unstable_fontFaces.ts

```diff
@@ -1,145 +1,123 @@
 import * as CSS from 'csstype';
 
+/**
+ * A small, internal utility to generate a standard `src` property value for `@font-face` declarations.
+ *
+ * MAINTAINER NOTE: Whenever an expected asset file changes (in this case a font file), the version number must be incremented. Otherwise, browsers may use the previously cached file for a long time.
+ */
+const buildSrc = (filename: string) =>
+  `url(/pds-assets-v1/fonts/${filename}.woff2) format("woff2")`;
+
```

Comment:
Good addition. Worth making a note in the Readme I think.

Reply:
Yeah it's on my list for the README, for right now, heads-up in the changelog for affected versions is fine.